pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -340,8 +347,13 @@ func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 			return nil, err
 		}
 
+		deviceScores, err := deviceScore(task.Pod, nodeInfo, weight.deviceScoreWeight)
+		if err != nil {
+			return nil, err
+		}
+
 		for _, node := range nodes {
-			nodeScores[node.Name] = podAffinityScores[node.Name] + nodeTolerationScores[node.Name] + podTopologySpreadScores[node.Name] + selectorSpreadScores[node.Name]
+			nodeScores[node.Name] = podAffinityScores[node.Name] + nodeTolerationScores[node.Name] + podTopologySpreadScores[node.Name] + selectorSpreadScores[node.Name] + deviceScores[node.Name]
```

Comment:
Why put device score in batchNodeOrder?

Reply:
because a node may need the information of all available nodes in order to score accurately

Reply:
Please make it score in parallel: )

Reply:
ok